socialdistribution/app/templatetags/filters.py

```diff
@@ -23,4 +23,12 @@ def get_nav(value):
 
 @register.filter(name="encode_url")
 def encode_url(value):
-    return urllib.parse.quote(value)
\ No newline at end of file
+    return urllib.parse.quote(value)
+
+@register.filter
+def get_post_id(url):
+    """
+    gets the post id from the comment page url
+    """
+    print(url.get_full_path())
```

Comment:
Is this print necessary, or is it for debugging purposes? We can leave it in if you want it to be logged